### PR TITLE
Add general meeting minutes for 2016-09-09

### PR DESCRIPTION
**Pull request by jakirkham**
This is a markdown converted copy of the meeting minutes from the general meeting on 2016-09-09 for archival. Please let me know if there need to be any changes. Will merge in 24hrs if there are no changes.

In attendance: @jjhelmus @ocefpaf @msarahan @mingwandroid @ericdill @bgruening @mwcraig

### CODE REVIEW

File: minutes/2016-09-09.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
+## 2016-09-09: General discussion
+
+Time: 14:00 UTC
+
+Hangout link: [](https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie)[https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie](https://hangouts.google.com/call/v5olhwzpfzgzpoq5i3wthjpqpie)
+
+**Attendees**
+
+*    Jonathan Helmus, Filipe, Michael, Ray, [Eric Dill](https://conda-forge.hackpad.com/ep/profile/yJqDqpPqJyz), Bjˆrn Gr¸ning, [Matt Craig](https://conda-forge.hackpad.com/ep/profile/yBvjHx0Ad3Y) (late)
```
Comment (mwcraig) [inline minutes/2016-09-09.md]: Looks like Bjorn's name got mangled...

Comment (bgruening) [inline minutes/2016-09-09.md]: `Björn Grüning` if you want to copy and paste it :)

File: minutes/2016-09-09.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+*   Minimum OSX required for clang (10.8, I think?)
+
+*   Actually clang is usable beginning in 10.7. So, this would be viable given your compatibility constraints.
+*   Also, all the refs I have seen suggest that this will still have C++11 support.
+
+*   Compatibility  with defaults (built on 10.7, uses gcc) - where will people break?  I  think only if mixing packages - how do we assure that we have all the  ones we need?
+
+*   Metadata unification with Continuum - are we OK with adding some fields to about section to match Anaconda standard?
+
+*   example at [](https://github.com/ContinuumIO/anaconda-recipes/blob/master/colander/meta.yaml)[https://github.com/ContinuumIO/anaconda-recipes/blob/master/colander/meta.yaml](https://github.com/ContinuumIO/anaconda-recipes/blob/master/colander/meta.yaml)
```
Comment (bgruening) [inline minutes/2016-09-09.md]: these points should be intended one level

File: minutes/2016-09-09.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+*   Can we add this to linter, and add to recipes as we update them?
+*   What support for unicode should we have?  Any?  Summary/description only?
+
+*   CUDA/cuDNN update
+
+*   Improving infrastructure
+
+        *   Better workflows with staged-recipes
+
+                *   Fast finish AppVeyor on merge ( [conda forge/staged recipes#1142](https://github.com/conda-forge/staged-recipes/pull/1142) )
```
Comment (bgruening) [inline minutes/2016-09-09.md]: indentation is wrong

File: minutes/2016-09-09.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+        *   Drop Travis CI matrix ( [conda forge/staged recipes#1234](https://github.com/conda-forge/staged-recipes/pull/1234) )
+        *   Use CircleCI for feedstock generation ( [conda forge/staged recipes#916](https://github.com/conda-forge/staged-recipes/issues/916) )
+        *   Keeping recipes out of PRs ( [conda forge/staged recipes#942](https://github.com/conda-forge/staged-recipes/issues/942) )
+        *   Bank work in partial conversion ( [conda forge/staged recipes#915](https://github.com/conda-forge/staged-recipes/issues/915) )
+
+*   Notifications (how do we stay on top of them)
+*   MSYS2
+
+        *   Available on defaults - was in conda 4.1.7, but that was pulled.  Coming in 4.1.8.
```
Comment (bgruening) [inline minutes/2016-09-09.md]: indentation

File: minutes/2016-09-09.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+*   Binary data
+
+        *   Do we include it in recipes?
+    *   What kinds do we allow if any (e.g. icons)?
+    *   How do we verify the licensing?
+    *   How do we verify that they are safe?
+
+*   Dev releases: Where do they happen?
+
+        *   Do we do them at conda-forge?
```
Comment (bgruening) [inline minutes/2016-09-09.md]: indentation

File: minutes/2016-09-09.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+*   Notifications (how do we stay on top of them)
+*   MSYS2
+
+        *   Available on defaults - was in conda 4.1.7, but that was pulled.  Coming in 4.1.8.
+    *   Discussing Ray Donnelly's work on MSYS2 packages and how we want to use and integrate these into conda-forge.
+    *   Some use cases to consider OpenBLAS, FFTW, build tools, others?
+
+*   Binary data
+
+        *   Do we include it in recipes?
```
Comment (bgruening) [inline minutes/2016-09-09.md]: here as well

**Comment by jakirkham** [comment]
Yeah I have tried to use Hackpad's Markdown download link, but it is actually pretty bad with indentation. Given Hackpad appears to be unsupported, we may need to think about upgrading to Dropbox's Paper to get better support.

File: minutes/2016-09-09.md (hunk 1 above)
Comment (jakirkham) [inline minutes/2016-09-09.md]: Yeah I guess Hackpad's markdown conversion struggles with unicode. Thanks for checking. Sorry your name got mangled, @bgruening. Will fix it.

Comment (jakirkham) [inline minutes/2016-09-09.md]: Corrected.

**Comment by jakirkham** [comment]
So I fixed the name mangling issue, but the formatting from Hackpad is a bit hopeless. I've tried to looking for alternative methods of export, but they all seem to rely on this same abysmal Hackpad mechanism. If someone can find a better way to do the export I'll try it, otherwise let's merge this as is. We can discuss moving to Dropbox's Paper next session.

**Comment by bgruening** [comment]
:+1:

**Comment by jakirkham** [comment]
Thanks guys.

**Pull request merged**